cloud/amazon/route53_zone.py

```diff
@@ -0,0 +1,148 @@
+#!/usr/bin/python
+
+DOCUMENTATION = '''
+module: route53_zone
+short_description: add or delete Route53 zones
+description:
+    - Creates and deletes Route53 private and public zones
+options:
+    zone:
+        description:
+            - The DNS zone record (eg: foo.com.)
+        required: true
+        default: null
+    command:
```

Comment:
Will do! I stole the `command` pattern from http://docs.ansible.com/route53_module.html. But I can easily change it to `state` :+1: